src/main.cpp

```diff
@@ -450,6 +573,40 @@ class gfxPipelineHandler {
         UpdateWindow(hWND);
         SetFocus(hWND);
     }
+
+    static void Install() {
+        InstallPatch("Enables pointer in windowed mode.", { 0x90, 0x90 }, {
+            0x4F136E,
+        });
+
+        if (!datArgParser::Get("16bit")) {
+            InstallPatch("Creates the initial window in 32-bit mode.", { 32 }, {
+                0x401475,
+                0x401477,
+            });
+        }
+
+        InstallCallback("gfxPipeline::SetRes", "Enables extra resolution parameters (e.g. '-window')",
```

Comment:
What wasn't working with it before? The call in main already allows use of the graphics args, so cdepth would enable 32 bit, and the call in BeginPhase uses the gfxConfig's settings, which have been modified to be 32 bit.